### PR TITLE
fix(stencil): use templates/ as the base directory if present

### PR DESCRIPTION
**Pull request by jaredallard**
## What this PR does / why we need it

Keeping templates in the root has been... a big mistake, so, instead, we'll place them in `templates/` and keep the behavior but treating that as a "chroot".

## Jira ID

[DTSS-0]

### CODE REVIEW

File: internal/codegen/stencil.go [hunk 1]
```diff
@@ -177,7 +177,14 @@ func (s *Stencil) getTemplates(ctx context.Context, log logrus.FieldLogger) ([]*
 		}
 
 		log.Debugf("Discovering templates from module %q", m.Name)
-		err = util.Walk(fs, "", func(path string, inf os.FileInfo, err error) error {
+		baseDir := ""
```
Comment (ChrisPRobinson) [inline internal/codegen/stencil.go:180]: Maybe good to offer a warning of this.

Comment (george-e-shaw-iv) [inline internal/codegen/stencil.go:180]:
```suggestion
		var baseDir string
```

Comment (jaredallard) [reply internal/codegen/stencil.go:180]: don't tell me how to LIVE MY LIFE

Comment (george-e-shaw-iv) [reply internal/codegen/stencil.go:180]: https://go-proverbs.github.io/

5th one down

**Review by george-e-shaw-iv — APPROVED** [review]

**Pull request merged**